### PR TITLE
added package command and descriptions table

### PR DESCRIPTION
**Pull request by sahandfarahani**
scshot

### CODE REVIEW

File: src/utils/tableGenerator.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export const tableGenerator = (headers: string[], data: object[]): string => {
+  let table = "";
+
+  headers.forEach((header) => (table += tagWrap(header, TH_TAG)));
+  table = tagWrap(table, TR_TAG);
+
+  data.forEach((line) => {
+    let row = "";
+    const keys = Object.keys(line);
+    keys.forEach((key) => {
```
Comment (ralph-dev) [inline src/utils/tableGenerator.ts:23]: Why not .map?
.map is better btw

File: src/utils/tableGenerator.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/**  generates an HTML table of out of given headers and data
+ * @param {string[]} headers - headers of table
+ * @param {object[]} data - data of table
+ * @returns {string} - HTML Table
+ */
+
+export const tableGenerator = (headers: string[], data: object[]): string => {
+  let table = "";
+
+  headers.forEach((header) => (table += tagWrap(header, TH_TAG)));
```
Comment (ralph-dev) [inline src/utils/tableGenerator.ts:17]: .map

File: src/utils/generateCommandTable.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+const NO_DATA = "";
+
+/**  generates a table of CoCommands and Descriptions from a package.json
+ * @param {string} readmeContent - content of the package.json
+ * @returns {string} - Table of Commands
+ */
+
+export const generateCommandTable = (readmeContent: string): string => {
+  const data = new Array();
+  readmeContent.match(COMMAND_LINE_REGEX)?.forEach((line) => {
```
Comment (ralph-dev) [inline src/utils/generateCommandTable.ts:19]: What's the ? For

**Review by ralph-dev — APPROVED** [review]
LGTM, just some super picky comments above, no need to fix just be aware

File: src/utils/generateCommandTable.ts (hunk 3 above)
Comment (sahandfarahani) [reply src/utils/generateCommandTable.ts:19]: @ralph-dev  there may not be any any command compatible to regex, so the object may become null

**Comment by ralph-dev** [comment]
Lgtm

**Pull request merged**